### ISSUE TITLE
404 for merged CSS

### ISSUE THREAD

**Issue by agorbulin** [fastly/fastly-magento2#174]
Hello.
We faced with broken styles on FE after flush caches.
**Info:**
- fastly cdn module 1.2.51
- magento ee 2.2.3, cloud
- fastly settings:
Purge category: Yes
Purge product: Yes	
Purge CMS page: Yes	
Preserve static assets on purge: No

**Steps to reproduce:**
1. Flush merged js/css files in Magento BO (button 'Flush JavaScript/CSS Cache')
2. Flush Magento caches in Magento BO (button 'Flush Magento Cache')
3. During flushing try to open several pages

**Expected results:**
Merged css/js cleaned, Fastly static content cache is flushed.

**Actual results:**
Merged css/js cleaned, Fastly static content cache is not flushed and as a result broken FE (see broken_fe.png) and 404 on css (see Merged_css_404.png)

broken_fe.png
![broken_fe](https://user-images.githubusercontent.com/36067568/39430848-9ea272a6-4c97-11e8-929c-7d1225d56eed.png)
Merged_css_404.png
![merged_css_404](https://user-images.githubusercontent.com/36067568/39430849-9ec24a9a-4c97-11e8-935e-14b743ab9b8d.png)

**Comment by deninchoo** [comment]
Hi.
I have tried to reproduce this issue but it doesn't seem like it is caused by the Fastly module. I followed the steps on different Fastly and Magento 2 versions, and indeed, the issue does appear in some cases, but it appeared regardless if Fastly is installed/enabled. This might be a general Magento 2 issue or even just an issue with the default theme. 

Does this issue still happen if Fastly is disabled? Does it happen on a non-default theme?
